tests/core/fixtures.py

```diff
@@ -229,7 +227,6 @@ def chain_without_block_validation():
     genesis_state = {
         funded_addr: {
             'balance': initial_balance,
-            'nonce': 0,
```

Comment:
It's main chain, so the nonce can't be removed.

Reply:
Sorry about that. Will fix it.